assets/css/style.css

```diff
@@ -0,0 +1,1003 @@
+}
+
+.deals-header-more:not(:first-of-type) {
+  margin-right: 23px;
+}
+
+.deals-header::after {
+  clear: both;
+  content: "";
+  display: block;
+}
+
+.deals-content {
+  display: flex;
+  flex: 0 1 auto;
+  flex-flow: row wrap;
+  margin: 25px 0;
+}
+
+.deal-content-container {
+  flex: 0 0 auto;
+  padding: 8px;
+  position: relative;
+  flex-basis: 100%;
+  max-width: 100%;
+  min-height: 60px;
+}
+
+.deal-content1 {
+  position: relative;
+  display: block;
+}
+
+.h4 {
+  color: #f42;
+  font-weight: 700;
+  line-height: 1.25;
+}
+
+.deal-content1 img {
+  width: 60px;
+  min-width: 60px;
+  max-width: 60px;
+  height: 60px;
+  min-height: 60px;
+  max-height: 60px;
+  margin: 10.6666666667px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+
+.deal-content1 > h4 {
+  background: #f42;
+  color: #fff;
+  font-weight: 700;
+  padding: 10.6666666667px 16px 10.6666666667px 84px;
+  margin: 0 !important;
+  font-size: 18px !important;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.deal-content-container h4 {
+  font-size: 16px;
+  margin-bottom: 12px;
+}
+
+@media (min-width: 1024px) {
+  .deal-content-container h4 {
+    font-family: Avalon, sans-serif;
+    font-size: 24px;
+    line-height: 1;
+  }
+}
+
+.deal-content1 div {
+  padding: 10.6666666667px 16px 10.6666666667px 84px;
+  color: #000;
+  background: rgba(255, 68, 34, 0.08);
+  margin: 0;
+}
+
+.deal-content1 div h4 {
+  font-size: 18px;
+  color: #000;
+  margin: 0 !important;
+  font-weight: 700;
+}
+
+.deal-content2 {
+  padding: 16px;
+}
+
+.deal-content3 {
+  font-size: 12px;
+  line-height: 18px;
+  padding: 16px;
+  border-top: 1px solid rgba(255, 68, 34, 0.25);
+  margin-top: 10px;
+  color: #555;
+  font-weight: 400;
+}
+
+.content {
+  display: flex;
+  flex: 0 1 auto;
+  flex-flow: row wrap;
+  margin: 25px 0;
+}
+
+.footer {
+  height: 257px;
+  margin-top: 50px;
+  text-align: center;
+}
+
+.footer-1 {
+  height: 70%;
+  background: #131313;
+}
+
+.footer-1 div {
+  padding: 18px;
+}
+
+.footer-social ul li {
+  color: white;
+  margin: 0 8px;
+  background: #333;
+  border-radius: 50%;
+  display: inline-block;
+  height: 40px;
+  padding: 12px;
+  width: 40px;
+}
+
+.footer-menu ul li {
+  color: #aaa;
+  font-family: Shentox, sans-serif;
+  font-size: 13px;
+  font-stretch: 100%;
+  font-weight: 500;
+  margin: 0 8px;
+  display: inline-block;
+}
+
+.footer-2 {
+  height: 30%;
+  background: #000;
+}
+
+.footer-2-wrapper {
+  display: flex;
+  align-items: baseline;
+  width: 570px;
+  margin: 0 auto;
+}
+
+.footer-2 p {
+  color: rgb(170, 170, 170);
+  display: block;
+  font-family: "ARS Maquette", "Helvetica Neue", Arial, sans-serif;
+  font-size: 13px;
+  text-align: center;
+  padding-top: 30px;
+}
+
+.footer-2 img {
+  height: 9px;
+  width: 66px;
+}
+
+.content1 {
+  flex: 0 0 auto;
+  padding: 8px;
+  position: relative;
+  flex-basis: 100%;
+  max-width: 100%;
+}
+
+@media (min-width: 768px) {
```

Comment:
[OPTIONAL]
This rule (`@media (min-width: 768px)`) is repeated 19x all over your code, it would be nice if you keep your code DRY and wrap all the rules related to that breakpoint in only one `@media (min-width: 768px)` declaration